hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -335,8 +335,11 @@ public boolean addSCM(AddSCMRequest request) throws IOException {
       }
       return raftClientReply.isSuccess();
     } catch (IOException e) {
-      LOG.error("Failed to update Ratis configuration and add new peer. " +
-          "Cannot add new SCM: {}.", scm.getScmId(), e);
+      LOG.warn("Failed to update Ratis configuration and add new peer. " +
+          "Cannot add new SCM: {}. {}", scm.getScmId(), e.getMessage());
+      if (LOG.isDebugEnabled()) {
+        LOG.info("Cannot add new SCM: {}. {}", scm.getScmId(), e);
```

Comment:
May be you want to print in debug and not info.
i) Also `if (LOG.isDebugEnabled())` is not required.
ii) No need to duplicate same log in warn and debug . `"Cannot add new SCM: {}. {}", scm.getScmId()`

Reply:
Thank you for the review! I have changed it to debug log. As we want to print only the stackTrace as apart of the debug log, I have added it to the LOG.debug. Could you please review it again?